packages/wonder-blocks-popover/__tests__/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -1575,6 +1577,76 @@ exports[`wonder-blocks-popover example 10 1`] = `
       }
     }
   >
+    <button
+      aria-label="Close Popover"
+      className=""
+      data-test-id="example-popover-close-btn"
```

Comment:
@jandrade modifying the example in `popover-content-core.md` modified the snapshot here. I think that the snapshot you noted above is for an example in `docs.md`.

Reply:
oh ok. Thanks for making this change!